common/beerocks/bwl/dwpal/ap_wlan_hal_dwpal.cpp

```diff
@@ -1675,6 +1675,34 @@ bool ap_wlan_hal_dwpal::generate_connected_clients_events()
     return ret;
 }
 
+bool ap_wlan_hal_dwpal::start_wps_pbc()
+{
+    refresh_vaps_info(beerocks::IFACE_RADIO_ID);
+    // start wps on the first fronthaul vap
```

Comment:
I think it's worth to explain in the commit message that specifically dwpal doesn't support WPS on multiple interfaces, so we take the first fronthaul interface.